tools/serverpod_cli/lib/src/logger/loggers/std_out_logger.dart

```diff
@@ -212,3 +225,30 @@ String _formatAsBox({
 
   return buffer.toString();
 }
+
+abstract class _SeveritySpanHelpers {
+  static LogLevel severityToLogLevel(SourceSpanSeverity severity) {
+    switch (severity) {
+      case SourceSpanSeverity.error:
+        return LogLevel.error;
+      case SourceSpanSeverity.warning:
+        return LogLevel.warning;
+      case SourceSpanSeverity.info:
+      case SourceSpanSeverity.hint:
+        return LogLevel.info;
+    }
+  }
+
+  static Object? highlightColor(SourceSpanSeverity severity) {
```

Comment:
Create constants for these.

Reply:
Double check that colors are readable for both dark/light.

Reply:
Looks good in both dark and light:
<img width="823" alt="Screenshot 2023-07-18 at 14 23 47" src="https://github.com/serverpod/serverpod/assets/137198655/ba3dc294-be6d-4370-9f33-9aaed17c9533">
<img width="819" alt="Screenshot 2023-07-18 at 14 24 04" src="https://github.com/serverpod/serverpod/assets/137198655/461eef68-c807-4184-b599-404ebb626121">